omod/src/main/java/org/openmrs/module/appui/rest/SessionController.java

```diff
@@ -12,12 +15,12 @@
 import org.openmrs.module.webservices.rest.web.representation.Representation;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.*;
```

Comment:
@dkayiwa
It was automatically made by intellij and I didn't check it. Sory for that.